ymir/backend/src/ymir_app/app/api/api_v1/endpoints/inferences.py

```diff
@@ -46,7 +46,7 @@ def call_inference(
 
     try:
         asset_dir, filename_mapping = save_files(inference_in.image_urls, settings.SHARED_DATA_DIR)
-    except FailedToDownload:
+    except (FailedToDownload, FileNotFoundError):
         logger.error("Failed to download user content: %s", inference_in.image_urls)
```

Comment:
Failed to download user content, or local file not found

Reply:
updated